src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -109,7 +109,9 @@ public void execute_appointmentUpdate_success() {
         showPersonAtIndex(model, INDEX_FIRST_PERSON);
 
         Person personInFilteredList = model.getFilteredPersonList().get(INDEX_FIRST_PERSON.getZeroBased());
-        Person editedPerson = new PersonBuilder(personInFilteredList).withAppointments(VALID_APPOINTMENT_ONE).build();
+        Person editedPerson = new PersonBuilder(personInFilteredList)
+                .withAppointments(VALID_APPOINTMENT_ONE).withAppointment("11/11/2025 1200")
```

Comment:
I might be missing something, but I'm a bit confused why there is a withAppointments and withAppointment. Does withAppointment do something that withAppointments didn't? 

If they have different functions, the naming might get a bit confusing

Reply:
withAppointments take in a Set, whereas withAppointment takes in a string datetime to add on to the existing Set.

Reply:
Should I rename it as addAppointment instead?

Reply:
Ahh ok, yeah that might be better to avoid any future accidents, thank you!